mantidproject/mantid
# Fix Qt3Support warnings group 5

This is group 5 of #16128.

This fixes the Qt3Support warnings for group 5.
## To test:

The changes in this PR have touched many areas of MantidPlot.  Below is a list of items which should be checked. Areas that need thorough testing:
- Drag a workspace into the main Mantid area
  - Confirm that is displayed
- Check that File->New works
  - Check the sub items, ie creating new table, matrix, note, etc. Create them and make sure that custom menus are created when bringing them into focus.
- Right click on Matrix, Plot, Table
  - Check different sub options
  - For plot: normalization options, waterfall plot, exportin of layer and window, color bar (lin+log, axes types, copy and past window

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
